project/threads/models.py

```diff
@@ -13,6 +13,17 @@
 from taggit.managers import TaggableManager
 from django.core.serializers.json import DjangoJSONEncoder
 
+class Category(models.Model):
+    name = models.CharField(max_length=63)
+
+    created = models.DateTimeField(auto_now_add=True, blank=True, null=True)
+    last_modified = models.DateTimeField(auto_now=True, blank=True, null=True)
+
+    def __str__(self):
+        return self.name
+
+    def __unicode__(self):
+        return self.name
```

Comment:
The `Category` model should not be moved as part of this pull request. There is a separate task to migrate the `Category` model to a `categories` app #1040.
```suggestion
```